utils/conversions.ts

```diff
@@ -22,10 +23,44 @@ export function apiServiceToService(service: ApiService): Service {
     organizationName: service.organization_name,
     requireSecureChannels: service.require_secure_channels,
     serviceId: service.service_id,
+    serviceMetadata: service.service_metadata && {
+      address: service.service_metadata.address,
+      appAndroid: service.service_metadata.app_android,
+      appIos: service.service_metadata.app_ios,
+      description: service.service_metadata.description,
+      email: service.service_metadata.email,
+      pec: service.service_metadata.pec,
+      phone: service.service_metadata.phone,
+      privacyUrl: service.service_metadata.privacy_url,
+      scope: service.service_metadata.scope,
+      tosUrl: service.service_metadata.tos_url,
+      webUrl: service.service_metadata.web_url
+    },
     serviceName: service.service_name
   };
 }
 
+// Returns an API Service Metadata from an internal Service model
+function getApiServiceMetadataFromRetrievedService(
+  retrievedService: RetrievedService
+): ApiServiceMetadata {
+  return (
+    retrievedService.serviceMetadata && {
```

Comment:
this method returns the boolean "false" in case retrievedService.serviceMetadata is undefined, you should use a ternary operator here